bundles/framework/myplaces3/MyPlacesTab.js

```diff
@@ -94,36 +97,22 @@ Oskari.clazz.define('Oskari.mapframework.bundle.myplaces3.MyPlacesTab',
                     panel.getContainer().empty();
                     panel.grid.renderTo(panel.getContainer());
 
-                    var editLink = this.linkTemplate.clone();
-                    editLink.addClass('categoryOp');
-                    editLink.addClass('edit');
-                    editLink.append(this.loc('tab.editCategory'));
-                    editLink.on('click', () => {
-                        sandbox.request(this.instance, editReqBuilder(categoryId));
-                        return false;
-                    });
-                    panel.getContainer().append(editLink);
+                    const modalWrapper = jQuery('<div class="myplaces-modal-wrapper"></div>');
+                    panel.getContainer().append(modalWrapper);
 
-                    var deleteLink = this.linkTemplate.clone();
-                    deleteLink.addClass('categoryOp');
-                    deleteLink.addClass('delete');
-                    deleteLink.append(this.loc('tab.deleteCategory'));
-                    deleteLink.on('click', () => {
-                        sandbox.request(this.instance, deleteReqBuilder(categoryId));
-                        return false;
-                    });
-                    panel.getContainer().append(deleteLink);
+                    const values = categoryHandler.getCategory(categoryId);
+                    const container = jQuery(modalWrapper)[0];
 
-                    const exportLink = this.linkTemplate.clone();
-                    exportLink.addClass('categoryOp');
-                    exportLink.addClass('export');
-                    exportLink.append(this.loc('tab.export.title'));
-                    exportLink.attr('title', this.loc('tab.export.tooltip'));
-                    exportLink.on('click', () => {
-                        window.location.href = this.instance.getService().getExportCategoryUrl(categoryId);
-                        return false;
-                    });
-                    panel.getContainer().append(exportLink);
+                    ReactDOM.render(
+                        <MyPlacesStyleForm
+                            layer={{ ...values, categoryId: categoryId }}
+                            visibility={ true }
```

Comment:
Do you need visibility prop if it's always true?

Reply:
Removed this. Left it there by mistake.